packages.json

```diff
@@ -13,6 +13,8 @@
 				"gnome-shell-extension-dash-to-dock",
 				"gnome-shell-extension-gsconnect",
 				"gnome-shell-extension-tailscale-status",
+				"gnome-shell-extension-logo-menu",
```

Comment:
Is this compatible with f38 or should we maybe put this in the f39 only section? Because if we ship this from the staging copr, we probably ship a f39 (gnome 45) only version.

Reply:
This is f38 compatible. this was done by building two different spec files and then being careful to make sure future builds were marked as f39 and newer only

Reply:
Perfect!

Reply:
Added a version gate so it'll build from the last commit that worked on GNOME 44, that also eliminates the need to be careful about build targets
https://github.com/ublue-os/Logomenu/commit/6cfa6aca55c3c6ea1f9911577123d8c9eb4ebad3